config/static.c

```diff
@@ -26,103 +35,121 @@
 #include "gatekeeper_lls.h"
 #include "gatekeeper_rt.h"
 
+#define RTE_LOGTYPE_CONFIG RTE_LOGTYPE_USER2
+
+/* TODO Get the install-path via Makefile. */
+#define LUA_BASE_DIR "./lua/"
+#define LUA_CONFIG_FILE     "gatekeeper_config.lua"
+
+struct gatekeeper_config {
+	struct bp_config 	*bp_conf;
+	struct catcher_config 	*catcher_conf;
+	struct dynamic_config 	*dy_conf;
+	struct cps_config 	*cps_conf;
+	struct ggu_config 	*ggu_conf;
+	struct gk_config 	*gk_conf;
+	struct gt_config 	*gt_conf;
+	struct rt_config 	*rt_conf;
+};
+
+static lua_State *lua_state;
```

Comment:
Sorry for the confusion. I understand that each instance can destroy its own resources when ``exiting = true``. My concern is that there may be some states shared by multiple instances/functional blocks, then who should be responsible for destroying these shared states? If that's not the case, the current code is enough.